### PR TITLE
VR: fix logging is not working and logs are not appended to /var/log/cloud.log

### PR DESCRIPTION
**Pull request by ustcweizhou**
## Description

There are no logs in /var/log/cloud.log when something are changed in VRs (for example start vms, add/remove rules).

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

**Comment by DaanHoogland** [comment]
code looks good @weizhouapache . Are we to understand that any modules imported don't get the changed logging config if it is set after importing?

**Comment by weizhouapache** [comment]
> code looks good @weizhouapache . Are we to understand that any modules imported don't get the changed logging config if it is set after importing?

@DaanHoogland yes. actually all python scripts in /opt/cloud/bin/cs does not have logging settings, so there is no log appended to /var/log/cloud.log
update_config.py is the entry to update VR configurations (including iptables rules/nics/configs,etc). 
we have similar code in elder versions which work fine. After systemvm refactoring by @rhtyd , it does not work any more.  I have no idea what caused it.

**Review by DaanHoogland — APPROVED** [review]

**Review by rhtyd — APPROVED** [review]

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
pylint doesn't approve of this, @weizhouapache @rhtyd (see 1st travis job) maybe we can mashup the order some more

File: systemvm/debian/opt/cloud/bin/update_config.py [hunk 1]
```diff
@@ -16,18 +16,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import logging
+logging.basicConfig(filename='/var/log/cloud.log', level=logging.INFO, format='%(asctime)s  %(filename)s %(funcName)s:%(lineno)d %(message)s')
+
```
Comment (DaanHoogland) [inline systemvm/debian/opt/cloud/bin/update_config.py:21]: not really helpfull, maybe. we can solve the pylint issue by tickering the order, something like:
```suggestion
import sys
import logging
import subprocess
import os
import os.path
import configure
import json

logging.basicConfig(filename='/var/log/cloud.log', level=logging.INFO, format='%(asctime)s  %(filename)s %(funcName)s:%(lineno)d %(message)s')

from subprocess import PIPE, STDOUT
```
Maybe with more `from x import y`'s. Or have some option on pylint.

**Comment by weizhouapache** [comment]
> pylint doesn't approve of this, @weizhouapache @rhtyd (see 1st travis job) maybe we can mashup the order some more

@DaanHoogland @rhtyd found another fix, tested ok. please review and test it

**Comment by rhtyd** [comment]
@blueorangutan package

**Review by rhtyd — APPROVED** [review]
Lgtm, but needs checking if we need to specify explicit log file path

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
great @weizhouapache more elegant and the travis errors so far are unrelated. it's easy to overlook the package init possibility, thanks for this.

**Comment by DaanHoogland** [comment]
marking this as major as it stops operators from being able to do proper diagnosis on the system/network

**Review by davidjumani — APPROVED** [review]
LGTM
Able to see logs when rules added

Screenshot from 2020-11-20 14-53-01

**Pull request merged**